src/drivers/driver_zephyr.c

```diff
@@ -821,6 +831,8 @@ static void *wpa_drv_zep_init(void *ctx,
 		goto out;
 	}
 
+	if_ctx->scan_res2 = NULL;
+
```

Comment:
Do we need to do this (since os_zalloc is already being done for if_ctx above)?

Reply:
No need, just an explicit assignment.